student.rb

```diff
@@ -9,4 +11,9 @@ def initialize(classroom, age, name = 'Unknown', parent_permission: true)
   def paly_hooky
     "¯\(ツ)/¯"
   end
+
+  def classroom=(classroom)
```

Comment:
- [ ] kindly remove the `=` after the name of the method. use ` def classroom(classroom)` instead of ` def classroom=(classroom)`

Reply:
@andersonlebon Thank you !!  can you provide the reason behind this request?